src/sentry/api/bases/organization.py

```diff
@@ -54,6 +54,14 @@ def has_object_permission(self, request, view, organization):
         return any(request.access.has_scope(s) for s in allowed_scopes)
 
 
+class OrganizationEventPermission(OrganizationPermission):
+    scope_map = {
+        'GET': ['event:read', 'event:write', 'event:admin'],
+        'POST': ['event:write', 'event:admin'],
+        'PUT': ['event:write', 'event:admin'],
+        'DELETE': ['event:admin'],
+    }
```

Comment:
Looks good. This matches the permissions set in the `ProjectEventPermission` class which is used by the ProjectGroupIndexEndpoint.